fetchai/ledger/bitvector.py

```diff
@@ -2,31 +2,49 @@
 
 
 class BitVector:
-
+    
     @staticmethod
     def from_bytes(data: bytes, bit_size: int):
-
-        # ensure the bit size matches the expectation
+        """
+            Takes a sequence of bytes, packs it into a BitVector object and returns the BitVector.
+            
+            :param data: a sequence of bytes in bytes object form.
+            :param bit_size: an integer representing the BitVector() size.
+            :return: the BitVector object containing the buffered data.
+            """
```

Comment:
Looks good @katharine-murphy  - The indentation is a bit weird here though

Reply:
Thanks, yes, I was swapping editors around so will avoid that in future :)